docs/api/conformance/Intents-Tests.md

```diff
@@ -0,0 +1,244 @@
+|------|-----------------------------|---------------------------------------------------------------------------------------------------|
+| Test | 1. Raise Intent             | Test raises an intent with `fdc3.raiseIntent("sharedTestingIntent2",testContextY,{appId:"<E's-appId>"})`<br />starts app E. |
+| E    | 2. Receive Intent & Context | After starting up, E runs `fdc3.addIntentListener("sharedTestingIntent2")` to register its listener.<br />It them receives `testContextY`, matching that sent by Test |
+| Test | 3. IntentResolution         | The `raiseIntent` call returns an `IntentResolution` Object with an `AppIdentifier` as the `source field` with App E's `appId` and `instanceId` set.   |
+| Test | 4. await results            | Test should `await resolution.getResult()` on the `IntentResolution` object returned in the previous step. A promise should be returned quickly.  |
+| E    | 5. return Channel           | E should retrieve a Channel object via `fdc3.getOrCreateChannel("someChannelName")` and return it immediately. |
+| Test | 6. receive Channel result   | The promise received by Test from `resolution.getResult()` should resolve to a `Channel` object with the expected id. Confirm that the `type` of the Channel object is "app". |
+| Test | 7. addContextListener       | Add a context listener to the Channel object via `channelObj.addContextListener("testContextZ",handler)` |
+| E    | 8. broadcast context        | After a short delay (of a few seconds) E should broadcast a `testContextZ` context object over the channel, including an `id` field with a unique identifier set (e.g. a uuid). |
+| Test | 9. receive context          | Test should receive the context broadcast by E and confirm that it contains the expected `id` value. |
+
+- `2.0-RaiseIntentChannelResult`: Perform the above test
+- `2.0-RaiseIntentPrivateChannelResult`: Perform the above test, but:
+  - Substitute app F throughout - which returns a PrivateChannel result instead of channel.
+  - At step 5, the PrivateChannel should be created via`fdc3.createPrivateChannel()`.
+  - At step 6 confirm that the type of the channel is "private".
+
+## PrivateChannels cannot be accessed as app channels
+
+| App  | Step                                      | Details                                                                                           |
+|------|-------------------------------------------|---------------------------------------------------------------------------------------------------|
+| Test | 1. Create a private channel               | Test creates a PrivateChannel with `const privChan = await fdc3.createPrivateChannel()`. Confirm that the Channel has an `id`. |
+| Test | 2. Confirm private channel `id` is unique | Test creates a second PrivateChannel with `const privChan = await fdc3.createPrivateChannel();`. Confirm that the Channel has an `id` and that it is distinct from the first channel created. |
```

Comment:
Typo - should it not be `const privChan2` here? 

https://github.com/finos/FDC3-conformance-framework/blob/main/src/test/v2.0/advanced/fdc3.raiseIntent.ts#L87

Reply:
Agreed, fixed.

```suggestion
| Test | 2. Confirm private channel `id` is unique | Test creates a second PrivateChannel with `const privChan2 = await fdc3.createPrivateChannel();`. Confirm that the Channel has an `id` and that it is distinct from the first channel created. |
```